lib/cn/blk_list.h

```diff
@@ -9,21 +9,18 @@
 #include <hse_util/inttypes.h>
 #include <hse_util/hse_err.h>
 
-#include <hse_ikvdb/blk_list.h>
-
-struct mpool;
+struct blk_list;
+struct kvs_block;
 struct mblock_props;
+struct mpool;
 
 #define BLK_LIST_PRE_ALLOC 64
 
 merr_t
-abort_mblock(struct mpool *dataset, struct kvs_block *blk);
+delete_mblock(struct mpool *dataset, struct kvs_block *blk);
 
 void
-abort_mblocks(struct mpool *dataset, struct blk_list *blks);
-
-merr_t
-delete_mblock(struct mpool *dataset, struct kvs_block *blk);
+delete_mblocks(struct mpool *dataset, struct blk_list *blk);
```

Comment:
I think we've moved away from the "dataset" concept and now only have "mpools", right?

Reply:
yes, that's correct. s/dataset/mp